Formula/glibc.rb

```diff
@@ -152,12 +153,60 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add ld.so.conf (will be linked to HOMEBREW_PREFIX/etc/ld.so.conf)
+    (prefix/"etc/ld.so.conf").atomic_write <<~EOS
+      # This file is generated by Homebrew. Do not modify.
+      #{lib}  # ensure Homebrew Glibc always comes first
+      include #{etc}/ld.so.conf.d/*.conf
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf"
+
+    # Create ld.so.conf.d directories
+    (prefix/"etc/ld.so.conf.d").mkpath
```

Comment:
Can we double check that `brew` creates `etc/"ld.so.conf.d"` as a directory instead of just a symlink to `prefix/"etc/ld.so.conf.d"`?

Reply:
Doesn't Homebrew will automatically create directory `HOMEBREW_PREFIX/etc/ld.so.conf.d` and make symlinks for files (not dir) on `brew link`? There is several files in `prefix/etc/ld.so.conf.d/*`.

Reply:
No, that is determined by logic in `Keg#link`.

For example, if a formula has a subdirectory `lib/"foo"` where `foo` is a directory that contains multiple files, then, `HOMEBREW_PREFIX/"lib/foo"` is a symlink to `lib/"foo"`.

Reply:
`etc` and `var` are treated differently than other directories. I've tweaked it now to use `etc` rather than `prefix/"etc"`. Passing `--sysconfdir=#{etc}` allows glibc to find the right directory.